src/main/java/com/appium/utils/DesiredCapabilityBuilder.java

```diff
@@ -54,7 +54,8 @@ public DesiredCapabilities buildDesiredCapability(String platform,
                     if ("browserName".equals(caps) && "chrome".equals(values.toString())) {
                         flag[0] = true;
                         try {
-                            desiredCapabilities.setCapability("chromeDriverPort",availablePorts.getPort());
+                            desiredCapabilities.setCapability("chromeDriverPort",
+                                    availablePorts.getPort());
```

Comment:
Whats the difference here? Wonder whats the change here? @shridharkalagi

Reply:
@SrinivasanTarget styleCheck fix, the line was more than 100 chars

Reply:
Oh